### DISCUSSION TITLE
Is edgemesh cni working in progess?

### DISCUSSION THREAD

**Discussion by WLBF** [kubeedge/kubeedge#3561] [Q&A] [answered]
It's common that cloud node can not reach edge node behind nat. Due to prometheus stack adapt scrape model to pull data from exporters running on the nodes, It's barely possible useing prometheus stack mointering edge cluster. It would be great if edgemesh cni can provide cross-subnet pod communication ability. Key feature list of edgemesh taged this feature not available, but will be supported in subsequent versions. I just wondering is this feature working in progress, and when will this feature be delivered.

**Comment by Poorunga** [comment, marked as answer]
It is a very important capability to allow pods to communicate across subnets. I suggest that you submit a feature issue in the edgemesh repository so that we can discuss the feature better. I can't guarantee the delivery time of edgemesh-cni feature for now, maybe v1.11.0 :)
